frontend/packages/citizen-frontend/src/App.tsx

```diff
@@ -61,7 +60,6 @@ export default function App() {
               />
               <Route path="/" component={RedirectToEnduser} />
             </Switch>
-            <Footer />
```

Comment:
My first thought would be that this belongs here (but removing it is probably intentional and for a good reason).

Reply:
Yeah the issue is that the map page wants to render footer differently than other pages. Couldn't come up with a better way to solve that, but ideas are welcome